robocop/utils/misc.py

```diff
@@ -234,3 +234,42 @@ def last_non_empty_line(node):
         if not isinstance(child, EmptyLine):
             return child.lineno
     return node.lineno
+
+
+def remove_robot_vars(name):
```

Comment:
Wow, we have some nice IFology here 😄

Reply:
I didn't spent last week on learning algorithms and compelx data structures for no reason!

Reply:
Refactored a bit